### ISSUE TITLE
Unable to customize table with TypeScript

### ISSUE THREAD

**Issue by tjaskula** [DevExpress/devextreme-reactive#895]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### Current Behaviour
I have TypeScript error on the snippet below:

```
const TableComponentBase = ({ classes, ...restProps }) => (
  <Table.Table
    {...restProps}
    className={classes.tableStriped}
  />
);
```
`TS2559: Type '{ className: any; }' has no properties in common with type 'IntrinsicAttributes & { children?: ReactNode; }'.` 

so I cannot customize table with `tableComponent` property on `<Table />` 

### Expected Behaviour

It should work

### Your Environment

devextreme-reactive: 1.1.1
react: 16.2.0
typescript: 2.7.1
bootstrap: none
react-bootstrap: none
material-ui: yes

**Comment by franklixuefei** [comment]
I encountered the exact same problem. I cannot even use `style`. This is pretty urgent and a road blocker.

**Comment by tjaskula** [comment]
Yeah for me it's also a roadblocker. I cannot use the component if customization doesn't work with typescript

**Comment by gsobolev** [comment]
Hi!

The fix is available in the 1.1.2 release.
